packages/components/src/tooltip/test/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { shallow, mount } from 'enzyme';
+import { fireEvent, render, screen } from '@testing-library/react';
```

Comment:
Let's make sure to use `@testing-library/user-event` instead of `fireEvent`. See [this article](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-testing-libraryuser-event) for rationale described by the library author.

Reply:
Fixed by 19d029c 👍